parlai/mturk/tasks/woz/scenarios/book_ride_v0.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title": "Book Ride",
+  "name": "Book Ride",
+  "version": 1,
+  "api_names": ["book_ride"],
+  "db": "ride.json",
+  "schema_urls": { "book_ride": "https://i.imgur.com/XvoZOMC.png" },
+  "instructions": {
+    "User": {
+      "completion_requirements": ["You have booked a ride."],
+      "completion_questions": [
+        "...find you a suitable ride?",
+        "...stay calm and helpful throughout the dialogue?"
+      ],
+      "task_description": "You want a ride from Forbes and Murray to Broadway and Main. Your name is Mark."
+    },
+    "Wizard": {
+      "task_description": "AA ...",
```

Comment:
It seems like this has to be changed, too?

Reply:
This is automatically generated from the template file. What's wrong with it?

Reply:
The task description should not be "AA ...". But if it is generated, I guess we can adjust the template in a separate PR.